### ISSUE TITLE
Halver (NPC) Non Responsive for any path of Rank 2 Mission 3 (Multiple Nations)

### ISSUE THREAD

**Issue by Djcocha** [DarkstarProject/darkstar#933]
Hi, I am posting this on behalf of a few players new to DSOldSchool server, they are trying to do the rank missions and have made it as far as 2-3. One of them is from Bastok and chose the Windy>Sandy  route and has been held up at the part where he needs to talk to Halver before the dragon fight because he gets no response whatsoever. Tonight I had just helped another (From Sandoria this time)  got as far as Killing the dragon in Waughroon Shrine & obtaining Key Item from first NPC after that, next NPC in line was Halver and not only did Halver not respond, but the guys mission log was reset to 'No current Missions' with mission 2-1 being the last under Completed Missions, but he still had the post-dragon Key Item. He went to gate guard and reactivated mission and it gave him new Consul letters and everything as if he was to restart the mission from scratch.

I hope this can be any help in finding out what the problem is, sorry if it seems confusing; I am relaying information not actually seeing the specifics of what is happening first hand.

**Comment by Djcocha** [comment]
So it seems another person new to the Server (Also from Sandoria) just recently finished that mission, Had the same issue that the original person from Sandy mentioned and had to do the missions twice. Both paths from Sandoria lose Mission log after getting the Key Item so that may be unrelated to Halver he is just the next part of that path, the guy from Bastok is still stuck and getting no response.

**Comment by xdemolish** [comment]
halver's used for a ton of quests/missions, likely bugging out when player has met conditions for more than one of the checks in the script

**Comment by Djcocha** [comment]
So, the people from Sandoria I am just going to assume whatever was happening with that was coincidence Because it was deleting their mission log after the NPC before Halver, but when they run through mission a second time everything works.

As for the guy from Bastok, he has no current quests with Halver as the starting NPC, and I just made a mule and ran him through the Bastok missions the same way as the other guy but I did not get stuck. I've told him to just try to run through the mission again from the start and see if any NPC responds to him along the way. Sorry for the confusion & inconvenience.
